lib/parsers/livesplit.rb

```diff
@@ -85,6 +85,7 @@ def v1_6(xml, fast)
         raise "Not a proper LiveSplit run file"
       end
 
+      run[:history].unshift(run[:time]) unless fast || run[:time].in?(run[:history])
```

Comment:
Alright, if that's as far as the problem goes and there's not some specific feature that depends on PB being in history, I'd prefer to not do this. If PB isn't in history, then putting it there doesn't necessarily complete the incomplete information, and it likely makes it inaccurate. If real-world history looks like

    0 1 2 3 4 5 6 7

and LiveSplit's account of real-world history looks like

    5 6 7

and we turn it into

    2 5 6 7

then we're misleading people into thinking that that's a contiguous segment of runs when really there could have been any number of runs between the first and the second. The information is *slightly* more complete, but I'd argue much more misleading, considering histories in our API have always been contiguous.

A PB time not being in a history is shitty but simple to understand once you realize LiveSplit didn't always record histories. But a buttload of missing times immediately before and after the first element in history -- and a number of run history entries that's sometimes-but-not-always one element greater than the number of split history entries in the last split -- is not as simple to understand.